src/lib.rs

```diff
@@ -6,43 +6,53 @@ use std::{fs, process};
 
 use anyhow::{bail, ensure, Context, Result};
 use notify::{RecommendedWatcher, RecursiveMode, Watcher};
-use structopt::StructOpt;
+use structopt::{lazy_static, StructOpt};
 use wasm_bindgen_cli_support::Bindgen;
 
+pub fn get_metadata() -> &'static cargo_metadata::Metadata {
+    lazy_static::lazy_static! {
+        static ref METADATA: cargo_metadata::Metadata = cargo_metadata::MetadataCommand::new()
+            .exec()
+            .expect("cannot get crate's metadata");
+    }
+
+    &METADATA
+}
+
 #[derive(Debug, StructOpt)]
 pub struct Build {
```

Comment:
We also need a function `package(name: &str)` that returns a package from the metadata.